Volume/Supporting/Constants.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Constants.swift
+//  Volume
+//
+//  Created by Cameron Russell on 4/16/21.
+//  Copyright © 2021 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+
+struct Constants {
+    static let missionStatement = """
+    Created by Cornell AppDev, Volume aims to better connect student publications with student readers by making it easy and accessible to digest student created content.\n\nVolume believes in the unique power that lies in the sharing and distribution of content by students and for students to spark conversation, experiences, and connections that improve the collegiate experience.
+    """
```

Comment:
What's the point of a multi line string if you don't split it onto multiple lines?